Dockerfile

```diff
@@ -75,6 +75,10 @@ ENV RUN_PYTHON_SDK_IN_DEFAULT_ENVIRONMENT=1
 RUN pip install .[gcp,dataflow]
 COPY . .
 
+COPY --from=apache/beam_python3.9_sdk:2.52.0 /opt/apache/beam /opt/apache/beam
+ENTRYPOINT ["/opt/apache/beam/boot"]
```

Comment:
Do we plan to merge https://github.com/apple/axlearn/pull/384 eventually?

Reply:
I think I can either include those changes in this repo or quickly merge #384 first. However, #384 is blocked by the same pytest failure I mentioned in chat